tests/fv3_conf/cpld_bmark_tiled_run.IN

```diff
@@ -109,3 +109,22 @@ if [ $WRITE_DOPOST = .true. ]; then
   cp    ${PATHRT}/parm/postxconfig-NT_FH00.txt postxconfig-NT_FH00.txt
   cp    ${PATHRT}/parm/params_grib2_tbl_new params_grib2_tbl_new
 fi
+
+#merra2
+if [ $IAER = 1011 ]; then
+  for n in 01 02 03 04 05 06 07 08 09 10 11 12; do
+    cp @[INPUTDATA_ROOT]/FV3_input_data_INCCN_aeroclim/MERRA2/merra2.aerclim.2003-2014.m${n}.nc aeroclim.m${n}.nc
+  done
+  cp    @[INPUTDATA_ROOT]/FV3_input_data_INCCN_aeroclim/aer_data/LUTS/optics_BC.v1_3.dat  optics_BC.dat
+  cp    @[INPUTDATA_ROOT]/FV3_input_data_INCCN_aeroclim/aer_data/LUTS/optics_OC.v1_3.dat  optics_OC.dat
+  cp    @[INPUTDATA_ROOT]/FV3_input_data_INCCN_aeroclim/aer_data/LUTS/optics_DU.v15_3.dat optics_DU.dat
+  cp    @[INPUTDATA_ROOT]/FV3_input_data_INCCN_aeroclim/aer_data/LUTS/optics_SS.v3_3.dat  optics_SS.dat
+  cp    @[INPUTDATA_ROOT]/FV3_input_data_INCCN_aeroclim/aer_data/LUTS/optics_SU.v1_3.dat  optics_SU.dat
+fi
+
+#ugwpv1
+if [ $DO_UGWP_V1 = .true. ]; then
+  cp    @[INPUTDATA_ROOT]/FV3_input_data/ugwp_c384_tau.nc ./ugwp_limb_tau.nc
+  cp    @[INPUTDATA_ROOT]/FV3_input_datac384/INPUT_L127/oro_data_ls* ./INPUT
```

Comment:
So these oro_data_ls* files are meant to optionally over-write the oro_data.tileX.nc files which are copied in at LN 34-40?